com.unity.template-hd/Assets/SampleSceneAssets/Tutorials/Pages/LightProbes_03.asset

```diff
@@ -35,13 +37,43 @@ MonoBehaviour:
       m_TutorialButtonText: 
     - m_Type: 0
       Title:
-        m_Untranslated: Congratulations!
+        m_Untranslated: 
       Text:
-        m_Untranslated: "In this tutorial, you learned what Light Probes are, how
-          they are placed, and how to visualize them. \r\n\r\nFor more information,
-          check out the Light Probes <a href=\"https://docs.unity3d.com/Manual/LightProbes.html\">documentation</a>
-          and <a href=\"https://forum.unity.com/forums/global-illumination.85/\">forums.</a>\r\n\r\n<b>Note:</b>
-          You do not need to save your changes in order to follow other tutorials."
+        m_Untranslated: 'Unlike in the previous per-object lighting probe system,
+          individual probes can not be moved or deleted. This is a hard limitation
+          of the system. In cases where we would need to move a set of probes, we
+          could use virtual offset instead.
+
+
+          <b>Virtual offset</b> allows
+          us to move the capture position of light probes. It can be enabled globally
+          in the <b>Lighting</b> > <b>Probe Volumes</b> pannel, moving all probes
```

Comment:
'moving all the probes' could be confusing (probe can not move). Even if it's explained above that only the capture position can be moved, maybe we should rephrase this ? 
==> It can be enabled globally for all the probes inside geometry using the Lighting > Probe Volumes panel where we can configure a Search Distance for a potential better capture position.